src/Cache/LegacyLinkBlockCache.php

```diff
@@ -20,7 +20,7 @@
 
 namespace PrestaShop\Module\LinkList\Cache;
 
-use PrestaShop\PrestaShop\Core\Addon\Module\ModuleRepository;
+use PrestaShop\PrestaShop\Core\Module\ModuleRepository;
```

Comment:
I think that we can keep compatibility with previous PrestaShop version by creating `PrestaShop\PrestaShop\Core\Module\ModuleRepository` that extends `PrestaShop\PrestaShop\Core\Addon\Module\ModuleRepository` if it doesn't exit already. WDYT?

Reply:
Don't know if it's important to keep compatibility for this kind of module, but it's still a good idea

Reply:
@atomiix & @PierreRambaud : Two solutions : 
- We create a BC Break in `ps_linklist`
- We remove the BC Break in Core : PrestaShop/PrestaShop#28247

Reply:
Why not doing it in the module? 🤔

Reply:
I don't think overriding a service in native module is a good idea.